lib/activerecord-import/adapters/postgresql_adapter.rb

```diff
@@ -118,7 +118,7 @@ def sql_for_on_duplicate_key_ignore( table_name, *args ) # :nodoc:
 
   # Returns a generated ON CONFLICT DO UPDATE statement given the passed
   # in +args+.
-  def sql_for_on_duplicate_key_update( table_name, *args ) # :nodoc:
+  def sql_for_on_duplicate_key_update( table_name, locking_column, *args ) # :nodoc:
     arg, primary_key = args
```

Comment:
We can leave this method signature the same and just do:

`arg, primary_key, locking_column = args`